web-app/src/main/webapp/script.js

```diff
@@ -61,14 +60,16 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
 window.onload = () => {
   document.getElementById("form-container").style.display = "none";
```

Comment:
Let's pick either single or double quotes and then make that change?

Reply:
What does your formatter do?